recipes/trf/meta.yaml

```diff
@@ -0,0 +1,19 @@
+package:
+  name: trf
+  version: "4.07b"
+
+source:
+  fn: trf407b
+  url: https://tandem.bu.edu/trf/downloads/trf407b.linux64 # [linux]
+  url: https://tandem.bu.edu/trf/downloads/trf407b.macos64 # [osx]
```

Comment:
Will see with folks of my project if they are useful. Could be nice to add them anyway here! Thanks @jxtx